### ISSUE TITLE
invite_used_EVT.js

### ISSUE THREAD

**Issue by Supimpa** [dbm-network/mods#665]
Someone help!
Before executing this event, I would like to check if the bot is allowed to create an invitation or manage a server. Not to be showing a lot of error on the console.

This is in invite_used_EVT.js
```prolog
const onReady = Bot.onReady
    Bot.onReady = function (...params) {
      // To not over-consume memory.
      if (Bot.$evts['Invite Used']) {
        setTimeout(() => {
          Bot.bot.guilds.cache.forEach(g => {
            g.fetchInvites().then(invites => {
              guildInvites[g.id] = invites
            })
          })
        }, 1000)
        Bot.bot.on('guildMemberAdd', DBM.Events.inviteUsed)
        Bot.bot.on('inviteDelete', inv => {
          inv.guild.fetchInvites().then(invites => {
            guildInvites[inv.guild.id] = invites
          })
        })
        Bot.bot.on('inviteCreate', inv => {
          inv.guild.fetchInvites().then(invites => {
            guildInvites[inv.guild.id] = invites
          })
        })
      }
      onReady.apply(this, ...params)
    }
```

**Comment by TheMonDon** [comment]
This doesn't have anything to do with permissions of creating an invite on invite_used.

**Comment by TheMonDon** [comment]
Alright, my bad this is an actual issue.

**Comment by TheMonDon** [comment]
Fixed in #682

**Comment by Supimpa** [comment]
**Thanks.**
There is another error being shown on the console, when it cannot find the code. Is it possible to make it less terrifying?

```
(node:3108) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'c
ode' of undefined
    at C:\Users\Dude\Desktop\Bot-V12\events\invite_used_EVT.js:20:
53
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
(node:3108) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This
error originated either by throwing inside of an async function without a catch
block, or by rejecting a promise which was not handled with .catch(). To termina
te the node process on unhandled promise rejection, use the CLI flag `--unhandle
d-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejectio
ns_mode). (rejection id: 4)
```
